wp-content/themes/citylimits/homepages/template.php

```diff
@@ -0,0 +1,38 @@
+<?php
+	global $shown_ids;
+
+	$topstory = largo_home_single_top();
+	$shown_ids[] = $topstory->ID;
+
+	$featured_stories = largo_home_featured_stories( 3 );
+?>
+
+<div id="homepage-featured">
+	<?php
+		largo_render_template( 'partials/home', 'top', array( 'topstory' => $topstory ) );
+	?>
+	<div id="featured">
+		<?php
+			/**
+			 * Grab three featured posts, output two, check if there's a widget area, output third or widget
+			 */
+			$counter = 0;
+			foreach ( $featured_stories as $featured ) {
+				$counter++;
+				if ( $counter != 3 ) {
```

Comment:
@benlk Should this be updated to `$counter >= 3` in case there are more than 3 posts marked as featured? Or does `$featured_stories` only include the posts about to be displayed?